x-pack/plugins/synthetics/public/apps/synthetics/components/common/components/monitor_status.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { EuiBadge, EuiDescriptionList, EuiLoadingSpinner } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { EncryptedSyntheticsMonitor } from '../../../../../../common/runtime_types';
+
+export const MonitorStatus = ({
+  loading,
+  monitor,
+  status,
+}: {
+  loading?: boolean;
+  monitor: EncryptedSyntheticsMonitor;
+  status?: 'up' | 'down';
+}) => {
+  const isBrowserType = monitor.type === 'browser';
+
+  const badge = loading ? (
+    <EuiLoadingSpinner size="s" />
+  ) : !status ? (
+    <EuiBadge color="default">{PENDING_LABEL}</EuiBadge>
+  ) : status === 'up' ? (
+    <EuiBadge color="success">{isBrowserType ? SUCCESS_LABEL : UP_LABEL}</EuiBadge>
+  ) : (
+    <EuiBadge color="danger">{isBrowserType ? FAILED_LABEL : DOWN_LABEL}</EuiBadge>
+  );
+
+  return <EuiDescriptionList listItems={[{ title: STATUS_LABEL, description: badge }]} />;
```

Comment:
We need to include these props to preserve the existing layout.

```suggestion
  return <EuiDescriptionList align="left" compressed listItems={[{ title: STATUS_LABEL, description: badge }]} />;
```

Note the title text is not uniform without it:

<img width="374" alt="image" src="https://user-images.githubusercontent.com/18429259/202009909-b5724926-680f-4297-a80d-844112ad5358.png">